src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableOrchestrationContext.cs

```diff
@@ -243,7 +243,21 @@ Task<DurableHttpResponse> IDurableOrchestrationContext.CallHttpAsync(HttpMethod
 
         async Task<DurableHttpResponse> IDurableOrchestrationContext.CallHttpAsync(DurableHttpRequest req)
         {
-            DurableHttpResponse durableHttpResponse = await this.ScheduleDurableHttpActivityAsync(req);
+            // calculate timeout expiration if DurableHttpRequest.Timeout is set
+            if (req.Timeout != null)
+            {
+                req.TimeoutExpirationDateTime = this.InnerContext.CurrentUtcDateTime + req.Timeout.Value;
+            }
+
+            DurableHttpResponse durableHttpResponse;
+            try
+            {
+                durableHttpResponse = await this.ScheduleDurableHttpActivityAsync(req);
+            }
+            catch (TimeoutException)
+            {
+                throw;
+            }
```

Comment:
There is no benefit to having a `catch` block that only does a `throw`. Let's remove the try/catch unless there is some way we can make it more useful.